src/components/app/App.scss

```
@@ -0,0 +1,8 @@
+@import '../../styles/colors';
```

добавь алиас `styles` и используй тут
```suggestion
@import 'styles/colors';
```

---

алиас?